apps/jobs/models.py

```diff
@@ -109,6 +111,7 @@ class Meta:
     age = models.PositiveIntegerField(default=None, null=True)
     education = models.TextField(max_length=500, default=None, null=True)
     professional_skills = models.TextField(max_length=500, default=None, null=True)
+    hiring_status = models.CharField(max_length=15, choices=HIRING_STATUS, default="N/A", null=True)
```

Comment:
Default type should not be "NA", It should be pending or "not applied yet".

Reply:
Replaced it with "Not Applied Yet"